rasa_nlu/extractors/duckling_extractor.py

```diff
@@ -50,7 +50,7 @@ def convert_duckling_format_to_rasa(matches):
         value = extract_value(match)
         entity = {"start": match["start"],
                   "end": match["end"],
-                  "text": match["text"],
+                  "text": match["body"],
```

Comment:
does this need fixing in the http extractor as well?

Reply:
nope 😄 https://github.com/RasaHQ/rasa_nlu/blob/master/rasa_nlu/extractors/duckling_http_extractor.py#L19